packages/desktop/components/AccountManagementDetails.svelte

```diff
@@ -42,7 +42,16 @@
     $: accountId = isAccountOutput(selectedOutput) ? (selectedOutput?.output as AccountOutput)?.accountId : null
     $: address = accountId ? getBech32AddressFromAddressTypes(new AccountAddress(accountId)) : null
     $: isMainAccount = accountId && accountId === $selectedWalletMainAccountId
+    $: balance = getAccountBalance(selectedOutput)
+    $: formattedBalance = formatTokenAmountBestMatch(balance, getBaseToken())
 
+    function getAccountBalance(outputData: OutputData): number | undefined {
+        if (isImplicitAccount) {
+            return Number(outputData.output.amount)
+        } else {
+            return undefined
```

Comment:
Add a TODO comment and create a new task to calculate the balance of an account output later 🙏🏼

Reply:
https://github.com/iotaledger/firefly/issues/8080